plugins/node/opentelemetry-instrumentation-express/README.md

```diff
@@ -78,6 +79,37 @@ Express instrumentation has few options available to choose from. You can set th
 - `info: ExpressRequestInfo` containing the incoming Express.js request, the current route handler creating a span and `ExpressLayerType` - the type of the handling layer or undefined when renaming the root HTTP instrumentation span.
 - `defaultName: string` - original name proposed by the instrumentation.
 
+#### Using `requestHook`
+
+Instrumentation configuration accepts a custom "hook" function which will be called on every request with the request handler span and the request information. Custom attributes can be set on the span or run any custom logic per request.
```

Comment:
> will be called on every request with the request handler span and the request information

Each request can generate multiple spans, right? It sounds from the text like this hook will be called once per request, but it will actually be called per middleware?
I suggest making it more clear.